### PR TITLE
Add RHEL 9.4 block for legacy module versions

### PR DESCRIPTION
**Pull request by Stringy**
## Description

Should fix the drivers failures on master.

### CODE REVIEW

File: kernel-modules/BLOCKLIST [hunk 1]
```diff
@@ -91,3 +91,4 @@
 # https://github.com/falcosecurity/libs/issues/918
 # Since kmods are not supported anyway, block these combination
 ~.*\.el8_9\..* * mod
+~.*\.el9_4\..* ~2\.[3-9]\..* *
```
Comment (Molter73) [inline kernel-modules/BLOCKLIST:94]: Do we want to expand this to block RHEL 9.5+ or should we wait for it to fail?

Comment (Stringy) [reply kernel-modules/BLOCKLIST:94]: Hmm I'm sort of on the fence; it's _probably_ not going to compile on RHEL 9.5+ but we might want build failures to tell us one way or another when it's released

Comment (Molter73) [reply kernel-modules/BLOCKLIST:94]: That's fine, just wanted to double check.

**Comment by Stringy** [comment]
can confirm the kernels are being blocked:

```
Blocked kernel: 5.14.0-427.13.1.el9_4.x86_64 2.3.0 mod
Blocked kernel: 5.14.0-427.13.1.el9_4.x86_64 2.3.0 bpf
Blocked kernel: 5.14.0-427.13.1.el9_4.x86_64+rt 2.3.0 mod
Blocked kernel: 5.14.0-427.13.1.el9_4.x86_64+rt 2.3.0 bpf
...
Blocked kernel: 5.14.0-427.13.1.el9_4.x86_64 2.5.0 mod
Blocked kernel: 5.14.0-427.13.1.el9_4.x86_64 2.5.0 bpf
Blocked kernel: 5.14.0-427.13.1.el9_4.x86_64+rt 2.5.0 mod
Blocked kernel: 5.14.0-427.13.1.el9_4.x86_64+rt 2.5.0 bpf
...
Blocked kernel: 5.14.0-427.13.1.el9_4.x86_64 2.6.0 mod
Blocked kernel: 5.14.0-427.13.1.el9_4.x86_64 2.6.0 bpf
Blocked kernel: 5.14.0-427.13.1.el9_4.x86_64+rt 2.6.0 mod
Blocked kernel: 5.14.0-427.13.1.el9_4.x86_64+rt 2.6.0 bpf
... etc
```

**Review by Molter73 — APPROVED** [review]

**Pull request merged**